src/Plugins/Loader/ManagedLoadContext.cs

```diff
@@ -175,7 +181,9 @@ protected override IntPtr LoadUnmanagedDll(string unmanagedDllName)
             var resolvedPath = _dependencyResolver.ResolveUnmanagedDllToPath(unmanagedDllName);
             if (!string.IsNullOrEmpty(resolvedPath) && File.Exists(resolvedPath))
             {
-                return LoadUnmanagedDllFromPath(resolvedPath);
+                return _loadInMemory
+                    ? LoadUnmanagedDllFromShadowCopy(resolvedPath)
+                    : LoadUnmanagedDllFromPath(resolvedPath);
             }
```

Comment:
Is there a reason why `LoadUnmanagedDllFromResolvedPath(string unmanagedDllPath)` was not used in this case?

Reply:
LoadUnmanagedDllFromResolvedPath was added as a workaround for https://github.com/natemcmaster/DotNetCorePlugins/issues/27. Basically, it just normalizes paths. In all cases I tested, the value of `resolvedPath` which is returned from `AssemblyDependencyResolver` is already normalized, so the normalization is unnecessary.

Reply:
The reason I'm asking is that if the condition for when shadow copying should be applied were to change, there are now two places one would have to update. If normalization of an already normalized path is fast, `LoadUnmanagedDllFromResolvedPath(string unmanagedDllPath)` could have been used for assemblies resolved by the native loader as well, to keep the shadow copy logic in one place.

An alternative would be to specify when normalization should be performed:
```csharp
private IntPtr LoadUnmanagedDllFromResolvedPath(
    string unmanagedDllPath, bool normalizePath = false)
{
    if (normalizePath)
    {
        unmanagedDllPath = Path.GetFullPath(unmanagedDllPath);
    }

    return _loadInMemory
        ? LoadUnmanagedDllFromShadowCopy(unmanagedDllPath)
        : LoadUnmanagedDllFromPath(unmanagedDllPath);
}
```

Reply:
I see. Whatever you think is best :) The way you have it is fine, and I'm also okay with the proposal in your comments.